context/context.go

```diff
@@ -15,25 +15,35 @@ import (
 	"github.com/pborman/uuid"
 )
 
+const MaxOpVersion = 40000
+const APIVersion = 1
+
 // Any object that is a part of the GlusterD context and needs to be available
 // to other packages should be declared here as exported global variables
 var (
-	MyUUID uuid.UUID
-	Rest   *rest.GDRest
-	TxnFw  *transaction.GDTxnFw
-	Store  *store.GDStore
+	MyUUID          uuid.UUID
+	Rest            *rest.GDRest
+	TxnFw           *transaction.GDTxnFw
+	Store           *store.GDStore
+	GlusterdVersion int
 )
 
 var (
 	initOnce sync.Once
 )
 
+func initOpVersion() {
+	//TODO : Need cluster awareness and then decide the op-version
+	GlusterdVersion = MaxOpVersion
```

Comment:
Ahh I got confused from the ReST API wiki, even I was also wondering why would we need to show op-version through /version end point. I'll correct it.